packages/airnode-examples/README.md

```diff
@@ -119,11 +119,12 @@ yarn create-aws-secrets
 
 ### 6. (Only if deploying to GCP) Create GCP credentials
 
-If you intend to deploy Airnode on GCP, you will need to sign in using the GCP CLI tool. If you are not sure how to do
-this or how to create a GCP account, see
+If you intend to deploy Airnode on GCP, you will need to create a service account for your project and add and download
```

Comment:
You've mentioned that I can use the previous auth mathod as well, but I am not sure how since it is not mentioned in the docs nor examples.

Reply:
I'd say that's fine. We're preferring this auth method and encourage users to use it. We're also not describing multiple auth methods you can use for AWS.